doc/source/examples/sharp-immersed-boundary-solver/cylinder-with-sharp-interface/cylinder-with-sharp-interface.rst

```diff
@@ -92,7 +92,7 @@ The initial condition has been modified compared to the initial solution propose
 The only thing that is left to define is the immersed boundary.
 .. note:: text
 
-	At this point, the immersed boundary tool only allows the representation of spherical objects. This means the immersed boundary can represent a circle in 2D and a spherical shell in 3d. We refer to the parameter definition of the immersed boundary in :doc:`../../../parameters/resolved-cfd-dem/resolved-cfd-dem` section of the documentation.
+	At this point, the immersed boundary tool only allows the representation of spherical objects. This means the immersed boundary can represent a circle in 2D and a spherical shell in 3d. We refer to the parameter definition of the immersed boundary in :doc:`../../../parameters/sharp-immersed-boundary-solver/sharp-immersed-boundary-solver` section of the documentation.
```

Comment:
What?
You introduced examples without spheres...???

Reply:
This is an old note it is wrong we have removed it.